006-connected-udp-sockets.md

```diff
@@ -0,0 +1,67 @@
+| Title  | Connected UDP sockets |
+|--------|-----------------------|
+| Author | @santigimeno          |
+| Status | DRAFT                 |
+| Date   | 2017-03-25 21:41:00   |
+
+
+## Overview
+
+Calling `connect()` on a UDP socket associates a remote address and port to the
+socket, so every message sent by this socket is automatically sent to that
+destination.
+
+## API
+
+The following functions are defined:
+
+```c
+int uv_udp_connect(uv_udp_t* handle, const struct sockaddr* addr);
+
+int uv_udp_disconnect(uv_udp_t* handle);
+
+int uv_udp_send(uv_udp_send_t* req,
+                uv_udp_t* handle,
+                const uv_buf_t bufs[],
+                unsigned int nbufs,
+                uv_udp_send_cb send_cb);
+
+int uv_udp_sendto(uv_udp_send_t* req,
```

Comment:
I'd like these APIs to remain unchanged, so how about not adding the new `sendto` versions and allowing for a NULL `addr`?